tfkbnufft/mri/dcomp_calc.py

```diff
@@ -82,3 +83,50 @@ def calculate_radial_dcomp_tf(interpob, nufftob_forw, nufftob_back, ktraj, stack
     )
 
     return dcomp
+
+
+def calculate_density_compensator(interpob, nufftob_forw, nufftob_back, ktraj, num_iterations=10):
+    """Numerical density compensation estimation for a any trajectory.
+
+    Estimates the density compensation function numerically using a NUFFT
+    interpolator operator and a k-space trajectory (ktraj).
+    This function implements Pipe et al
+
+    This function uses a nufft hyper parameter dictionary
+    and k-space trajectory.
```

Comment:
While I am at it, might as well:
```suggestion
    This function uses a nufft hyper parameter dictionary, the associated nufft
    operators and k-space trajectory.
```